images/haproxy-1.8/ROOT/srv/templates/haproxy-backend-http.cfg.tmpl

```diff
@@ -8,9 +8,12 @@ backend dotcms_http
     http-reuse aggressive
     option http-server-close
     option forceclose
-    cookie BACKENDID insert indirect nocache
-{{ with .Env.CMS_BACKEND_SERVERS }}{{ range $index, $server := split . "," }}    server dotcms_{{ $server }}_{{ $http_port }} {{ $server }}:{{ $http_port }} cookie {{ $server }} check
-{{ end }}{{ end }}
+    stick-table type string len 32 size 10000 expire 1h
```

Comment:
Moved to use a stick-table in haproxy - this prevents haproxy from having to set a cookie and instead relies on the JSESSIONID.

Having haproxy set a cookie forced the header `Cache-control: Private` on all requests and prevented browsers from being able to cache backend resources.